Dockerfile

```diff
@@ -78,7 +78,7 @@ FROM ${PKGS_PREFIX}/zlib:${PKGS} AS pkg-zlib
 FROM ${PKGS_PREFIX}/zstd:${PKGS} AS pkg-zstd
 
 # Installs tailwindcss
-FROM docker.io/oven/bun:1.2.4-alpine AS tailwind-base
+FROM --platform=${BUILDPLATFORM} docker.io/oven/bun:1.2.4-alpine AS tailwind-base
```

Comment:
thank you for your PR, but unfortunately this file is automatically generated via `kres`, so we need either a fix to 	`kres` or to `.kres.yaml`

Reply:
Good to know! I'll look into that when i got time.

Reply:
https://github.com/siderolabs/image-factory/pull/222/files#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557R81 should fix this

Reply:
@mattiascockburn thanks for your PR, I will close it as #222 contains the fix via `.kres.yaml` update